abm/connector.py

```diff
@@ -236,9 +218,22 @@ def read_stream(self, catalog_file):
 
         # step 3: Run the Airbyte read operation to read the datasets
         return self.run_container('read --config '
-                      + self.name_in_container(self.conf_file.name)
+                      + self.name_in_container(self.conf_file.name, MOUNTDIR)
                       + ' --catalog '
-                      + self.name_in_container(catalog_file.name))
+                      + self.name_in_container(catalog_file.name, MOUNTDIR))
+
+    '''
+    Given a catalog return the json schema of a specific stream (table) if the stream
+    is provided. Otherwise return the json schema of the first stream.
+    '''
+    def get_stream_schema(self, catalog):
```

Comment:
Maybe this method would no longer be needed once `self.json_schema` contains the schema for a single stream